plugins/ps/post-deploy

```diff
@@ -4,5 +4,8 @@ source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
 source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
+DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
+
+[[ -f "$DOKKU_PROCFILE" ]] && rm -f "$DOKKU_PROCFILE"
```

Comment:
Sorry I meant to say in the deploy phase. IIRC, you call `get_cmd_from_procfile()` in `dokku`'s deploy phase. Thus, perhaps we can do everything in one swing as opposed to extracting the `Procfile` in `pre-deploy`, using it in `deploy` and removing it in `post-deploy`.

What do you think?